src/init.cpp

```diff
@@ -230,9 +225,7 @@ void Shutdown()
         pblocktree = NULL;
     }
 #ifdef ENABLE_WALLET
```

Comment:
Because `walletinit.cpp` is in libbitcoin_wallet, so is not linked if `ENABLE_WALLET` isn't defined. #10762 will remove the libbitcoin_server -> libbitcoin_wallet dependencies entirely from init.cpp.